### PR TITLE
Fix broken command parameters

### PR DESCRIPTION
**Pull request by Flonja**

### CODE REVIEW

File: src/AvailableCommandsPacket.php [hunk 1]
```diff
@@ -145,9 +145,12 @@ public static function convertArg(int $protocolId, int $type) : int{
 		}
 
 		return match ($type) {
+			self::ARG_TYPE_TARGET => 0x07,
 			self::ARG_TYPE_WILDCARD_TARGET => 0x08,
+			self::ARG_TYPE_FILEPATH => 0x10,
 			self::ARG_TYPE_STRING => 0x20,
-			self::ARG_TYPE_POSITION => 0x28,
+			self::ARG_TYPE_INT_POSITION => 0x28,
```
Comment (Driesboy) [inline src/AvailableCommandsPacket.php:152]: If I'm not mistaken ARG_TYPE_INT_POSITION didn't exist before 1.18.30

**Review by Driesboy — CHANGES_REQUESTED** [review]

File: src/AvailableCommandsPacket.php (hunk 1 above)
Comment (Flonja) [reply src/AvailableCommandsPacket.php:152]: Isn't `ARG_TYPE_INT_POSITION` just a block position?

Comment (Driesboy) [reply src/AvailableCommandsPacket.php:152]: Well only one of them existed pre 1.18.30. I never worked with any of them, but I doubt they changed the name of a thing that already existed & added a new implementation with the old name

Comment (Flonja) [reply src/AvailableCommandsPacket.php:152]: Just tested `0x28` in protocol `475` and it definitely exists

Comment (Driesboy) [reply src/AvailableCommandsPacket.php:152]: 0x28, yes but 0x29 didn't

**Pull request merged**